modules/securing-container-images-by-using-signature-integration.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * operating/verify-image-signatures.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="securing-container-images-by-using-signature-integration_{context}"]
+= Securing container images by using signature integration
+
+By creating a signature integration, you can ensure that a  trusted source signs the container image. 
+
+When you create a signature integration, you can use the following verification methods:
```

Comment:
The transparency log integration is not a verification method on equal terms as public keys or certificates. Rather, the transparency log is used in combination with either public keys or certificates (transparency log cannot be used just by itself). It serves as an audit log and enhances the security of the former methods.